Resources/config/mongodb.php

```diff
@@ -0,0 +1,91 @@
+use Doctrine\ODM\MongoDB\Mapping\Driver\AttributeDriver;
+use Doctrine\ODM\MongoDB\Tools\ResolveTargetDocumentListener;
+use Doctrine\Persistence\Mapping\Driver\MappingDriverChain;
+use MongoDB\Client;
+use ProxyManager\Proxy\GhostObjectInterface;
+use Symfony\Bridge\Doctrine\ContainerAwareEventManager;
+use Symfony\Bridge\Doctrine\Security\User\EntityUserProvider;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->parameters()
+        ->set('doctrine_mongodb.odm.cache.array.class', 'Doctrine\Common\Cache\ArrayCache')
+        ->set('doctrine_mongodb.odm.cache.apc.class', 'Doctrine\Common\Cache\ApcCache')
+        ->set('doctrine_mongodb.odm.cache.apcu.class', 'Doctrine\Common\Cache\ApcuCache')
+        ->set('doctrine_mongodb.odm.cache.memcache.class', 'Doctrine\Common\Cache\MemcacheCache')
+        ->set('doctrine_mongodb.odm.cache.memcache_host', 'localhost')
+        ->set('doctrine_mongodb.odm.cache.memcache_port', 11211)
+        ->set('doctrine_mongodb.odm.cache.memcache_instance.class', 'Memcache')
+        ->set('doctrine_mongodb.odm.cache.xcache.class', 'Doctrine\Common\Cache\XcacheCache')
+        ->set('doctrine_mongodb.odm.connection.class', Client::class)
+        ->set('doctrine_mongodb.odm.configuration.class', Configuration::class)
+        ->set('doctrine_mongodb.odm.document_manager.class', DocumentManager::class)
+        ->set('doctrine_mongodb.odm.manager_configurator.class', ManagerConfigurator::class)
+        ->set('doctrine_mongodb.odm.class', ManagerRegistry::class)
+        ->set('doctrine_mongodb.odm.metadata.driver_chain.class', MappingDriverChain::class)
+        ->set('doctrine_mongodb.odm.metadata.attribute.class', AttributeDriver::class)
+        ->set('doctrine_mongodb.odm.metadata.xml.class', XmlDriver::class)
+        ->set('doctrine_mongodb.odm.mapping_dirs', [])
+        ->set('doctrine_mongodb.odm.xml_mapping_dirs', '%doctrine_mongodb.odm.mapping_dirs%')
+        ->set('doctrine_mongodb.odm.document_dirs', [])
+        ->set('doctrine_mongodb.odm.fixtures_dirs', []);
+
+    $containerConfigurator->services()
+
+        ->alias(DocumentManager::class, 'doctrine_mongodb.odm.document_manager')
+
+        ->alias(ManagerRegistry::class, 'doctrine_mongodb')
+
+        ->set('doctrine_mongodb.odm.connection.event_manager', ContainerAwareEventManager::class)
+            ->abstract(true)
+            ->args([
+                service('service_container'),
+            ])
+
+        ->set('doctrine_mongodb.odm.container_repository_factory', ContainerRepositoryFactory::class)
+            ->args([''])
+
+        ->set('doctrine_mongodb.odm.manager_configurator.abstract', ManagerConfigurator::class)
+            ->abstract(true)
+            ->args([
+                [],
```

Comment:
This argument is always replaced by the extension. You can use `abstract_arg`.

```suggestion
                abstract_arg('enabled filters'),
```